### ISSUE TITLE
Virus Total flags your own program as malware

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [SamuelTulach/VirusTotalUploader#83]
Please see [here](https://www.virustotal.com/gui/file/80b5d499cbc342e55d3575b0d9313bca5d14c9a3df9a2d8e4040d97589c75d6c/detection/f-80b5d499cbc342e55d3575b0d9313bca5d14c9a3df9a2d8e4040d97589c75d6c-1629891422), which is the VirusTotal report for the zipped version of your program. Obviously, this is 'doubleplus not good'.

**Comment by LinuxOnTheDesktop** [comment]
The situation seems [even worse](https://www.virustotal.com/gui/file/fb16d3b627ec47513599b239b8f37763ae6cdc24fcaa38204781c8cb72770f15/detection) (though VirusTotal appears unable to count . .) with the unzipped file.

**Comment by SamuelTulach** [comment]
Unfortunately, as it has been [discussed](https://github.com/SamuelTulach/VirusTotalUploader/issues/72) [numerous](https://github.com/SamuelTulach/VirusTotalUploader/issues/66) [times](https://github.com/SamuelTulach/VirusTotalUploader/issues/61), without code signing certificate, anti-viruses tend to flag it as risk ware.

That's why the executables you see in release section are built on AppVeyor, directly from source code.

**Comment by LinuxOnTheDesktop** [comment]
Presumably you should put something to that effect in the README.

**Comment by SamuelTulach** [comment]
After a quick check and chat with some people, it seems like the empty info in AssemblyInfo.cs [was causing the detections](https://www.virustotal.com/gui/file/2e6e5f4cb28051f29e3f82fcd939ff423862dc539bbd965a05df971d02a2756c/detection). Also I will hopefully have a code signing certificate for the future.

**Comment by SamuelTulach** [comment]
Unfortunately InnoSetup still seems to get flagged. I will probably try to move away from it, but that would need a custom installer since I am sure other generic install creators would be flagged.

**Comment by LinuxOnTheDesktop** [comment]
Right. In the meantime I do advise modifying the README. Thank you for your software.

**Comment by SamuelTulach** [comment]
I've got [code signing certificate now](https://github.com/SamuelTulach/VirusTotalUploader/releases/tag/0.1.9) too. Unfortunately the InnoSetup installer is still (ironically) getting flagged by some AVs. I will probably end up writing a custom minimal installer myself.

**Comment by vertigo220** [comment]
Considering they themselves mention and link to this software as a replacement for their own, discontinued, version, you'd think they would make the effort to make sure this didn't happen as well as ensure there is no malware in it. But I agree, if this is an issue, it's best to mention it in the readme and probably even on the releases page.
